prime-dotnet-webapi/Models/Enrollee.cs

```diff
@@ -31,6 +31,9 @@ public class Enrollee : BaseAuditable, IValidatableObject, IUserBoundModel
         [Required]
         public string LastName { get; set; }
 
+        [Required]
+        public string GivenNames { get; set; }
```

Comment:
I assumed we were replacing MiddleName with GivenNames

Reply:
Was going to ask about that, Not sure if we ever enter any data into the middle name field.

Reply:
Yah, we're replace middle name with given names.  The given names will only appear in the admin UI, and status quo during enrolment like you have it.

Reply:
👍